machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts

```diff
@@ -71,6 +71,7 @@ const model = createModel(
     walletBindingSuccess: false,
     publicKey: '',
     privateKey: '',
+    vcKey: '',
```

Comment:
two doubts 

1. won't vcKey will be same as vcKey in metadata. if it is, why to get it from store and store it separately in context instead of using vcMetadata
2. if it is not same vcKey, please name it appropriate to signify that this VcKey of tempered vc